vowpal_wabbit/ubuntu1604-build.Dockerfile

```diff
@@ -45,6 +45,17 @@ RUN version=3.13 && build=5 \
  && for filename in /opt/cmake/bin/*; do echo Registering $filename; ln -fs $filename /usr/local/bin/`basename $filename`; done \
  && rm cmake-$version.$build-Linux-x86_64.sh
 
+RUN wget https://github.com/google/flatbuffers/archive/v1.12.0.tar.gz \
+ && tar -xzf v1.12.0.tar.gz \
```

Comment:
can you bubble up the version number similar to how we do the cmake install? since the number is being referenced in multiple parts